engine/LICENSE

```diff
@@ -0,0 +1,191 @@
+                                 Apache License
```

Comment:
For the sake of go docs (go.pkg) let's keep the LICENSE file, in models, engine, spv-wallet, but ensure that we have MIT in all of them (so replace this Apache with MIT)